src/Components/Web/test/Virtualization/VirtualizeTest.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.Virtualization
+{
+    public class VirtualizeTest
```

Comment:
Is this different from `VirtualizationTest` or can it just be removed?

Reply:
I think there are some unit tests that might make sense, like testing that exceptions thrown in `ItemsProivder` are properly dispatched to the renderer and that supplying invalid arguments to `<Virtualize>` throws exceptions.